nrich-jackson/README.md

```diff
@@ -4,16 +4,18 @@
 
 ## Overview
 
-nrich-jackson is a library containing that adds additional Jacksons serializers and deserializers that have proved to be useful in projects
+`nrich-jackson` is a module containing that adds additional Jackson's serializers and deserializers that have proved to be useful in projects
 (i.e. serializing empty strings to null, deserializing class property with certain classes etc.)
 
 ## Setting up Spring beans
 
-Functionalities of this library are defined as separate Jackson modules. And are located in `JacksonModuleUtil` class. Modules can them self be declared as beans and will be found automatically when
-Spring Boot is used or they can be manullay registered on
-`ObjectMapper` instance.
+Functionalities of this module are defined as separate Jackson modules. And are located in `JacksonModuleUtil` class. Modules can them self be declared as beans and will be found automatically when
```

Comment:
`Functionalities of this module are defined as separate Jackson modules. And are located in JacksonModuleUtil class` - this should be one sentence. The sentence after this is not clear to me. I suppose it should be `Modules can be declared...` or something like that?